app/controllers/datasets_controller.rb

```diff
@@ -59,21 +68,75 @@ def resubmit
     redirect_to dataset_path(dataset)
   end
 
+  # Outputs the Datacite XML representation of the dataset
+  def datacite
+    @dataset = Dataset.find(params[:id])
+    work = Work.find(@dataset.work_id)
+    resource = Datacite::Resource.new_from_json(work.data_cite)
+    render xml: resource.to_xml
+  end
```

Comment:
This was a bonus feature, it outputs the Datacite XML for a given dataset, which turns out will be very beneficial for Matt and Neggin.

Reply:
💯